Remotion/Web/Test.Shared/IFrameSupport/FrameContentFunction.cs

```diff
@@ -29,6 +29,6 @@ public FrameContentFunction ()
     }
 
     [UsedImplicitly]
-    private WxeStep Step1 = new WxePageStep ("~/IFrameSupport/FrameContent.aspx");
+    private WxeStep Step1 = new WxeResourcePageStep ("IFrameSupport/FrameContent.aspx");
```

Comment:
Since we have a couple of recurring paths, both partial and complete, we could also introduce a factory:
ResourceFactory.CreatePageStepForIFrameSupportFrameContent();
_resourceUrlFactory.GetErrorHandlingErrorForm();
_resourceUrlFactory.GetStartForm();

It could also be okay, I guess, if the extension factory methods instead where just stand alone factory methods and resolve the ResourceUrlFactory on their own. That's just not something I like to do in production code due to the Composition Root thingy.

Reply:
I looked through the usages and could not find any real repetition within the construction. This line of code is only used once in the Web.Test project. Same with `ErrorHandling/ErrorForm.aspx`.

Introducing one for the `Start.aspx` would make a little more sense, with there being 5 usages, but then we also have differnt conventions in use for names of start forms `Default.aspx` vs `Start.aspx` vs `StartForm.aspx`

Reply:
Okay, it felt like there was repetition. Maybe there's just too many files with smiliar names.

Anyhow, if there's not enough repetition, we don't need the factory and I'd recomment just going to the first step, as discussed in the other change/comment.